src/bot/integrations/donationalerts.js

```diff
@@ -50,6 +50,8 @@ class Donationalerts {
         })
     } else this.socket.connect()
 
+    setTimeout(() => this.reconnect(), 60 * 60 * 2 * 1000) // restart socket each 2 hours
```

Comment:
setTimeout is doing only one restart, you want to use setInterval probably

Reply:
Also would like to see numbers reordered 2 * 60 * 60 * 1000, pretty dislike this milliseconds counting :D will probably add something into commons to be more readable like commons.time.hours(2) or something like that not sure

Reply:
On bot starting it calls connect() function, connect function calls setTImeout this.reconnect(), and this.reconnect() calls connect(), what call reconnect() again, doesnt right? if i'll do setInterval it will reconnects 99999 times e.t.c, doesnt right?

Reply:
Oh right, didn't notice that its in connect, so it should be ok